doc/api/diagnostics_channel.md

```diff
@@ -1103,14 +1103,10 @@ for the sync error and one for the async error.
 
 ### Built-in Channels
 
-> Stability: 1 - Experimental
-
-While the diagnostics\_channel API is now considered stable, the built-in
-channels currently available are not. Each channel must be declared stable
-independently.
-
```

Comment:
I was initially unsure whether this part should be removed, but after seeing the CI fail due to a duplicate stability status inherited from the parent (https://github.com/nodejs/node/actions/runs/14946879215/job/41993454082?pr=58261), it seems that it needs to be taken out.